atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -259,7 +259,8 @@ private TableReference getTableToSweep(SweepTransaction t,
     }
 
     private double getSweepPriority(SweepPriorityRowResult oldPriority, SweepPriorityRowResult newPriority) {
-        if (AtlasDbConstants.hiddenTables.contains(newPriority.getRowName().getFullTableName())) {
+        Set<String> hiddenTableFullNames = AtlasDbConstants.hiddenTables.stream().map(tableRef -> tableRef.getQualifiedName()).collect(Collectors.toSet());
```

Comment:
Cool. I'm still new to java 8 so keep telling me random other things I don't know as I try to wean myself off of guava's deprecated java8 dupes.